src/Files.Uwp/ViewModels/Widgets/Bundles/BundleItemViewModel.cs

```diff
@@ -45,7 +45,23 @@ public string Name
         {
             get
             {
-                string fileName = System.IO.Path.GetFileName(this.Path);
+                string fileName;
+
+                // Network Share path
+                if (System.IO.Path.GetPathRoot(this.Path) == this.Path && this.Path.StartsWith(@"\\")) 
```

Comment:
Are you sure? This will check if the path starts with double "\\", not single "\"

Reply:
Yep, UNC paths start with ''\\'' , that is what i am checking for there.
If `this.Path` it is a UNC path and has no subfolders.